app/models/grda_warehouse_base.rb

```diff
@@ -19,15 +20,6 @@ def self.postgres?
     connection.adapter_name.in?(['PostgreSQL', 'PostGIS'])
   end
 
-  def self.reset_connection
```

Comment:
seems unused

Reply:
I feel like this was used for some debugging at some point.  Happy to see it removed for now.